src/interface/middleware/canUpload.js

```diff
@@ -1,37 +1,33 @@
 const getStorageUse = require("../../domain/limit/getStorageUse");
 const ErrorHandler = require("../../infra/errorHandler");
 
-
-async function checkStorageLimit(contractAddress, invokerAddress) {
-  if (!contractAddress && !invokerAddress) {
-    return false;
-  }
-
-  const limit = await getStorageUse({ contractAddress, invokerAddress });
-  const totalAllowedStorage = limit.storageLimit + limit.extraStorage;
-
-  return limit.storageUse >= totalAllowedStorage;
-}
-
 async function canUpload(req, res, next) {
   const invokerAddress = req.invokerAddress;
   const contractAddress = req.contractAddress;
-  const namespace = req.namespace;
```

Comment:
In this just removing namespace should be fine imo and keeping the old refactored code as it much cleaner.

Reply:
alright. I thought so too, but went to reverting the code itself.